build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,109 @@
+
+# Enable AntreaIPAM, which can allocate IP addresses from IPPools. AntreaIPAM is required by the
+# bridging mode and allocates IPs to Pods in bridging mode. It is also required to use Antrea for
+# IPAM when configuring secondary network interfaces with Multus.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "AntreaIPAM" "default" false) }}
+
+# Enable multicast traffic.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "Multicast" "default" false) }}
+
+# Enable Antrea Multi-cluster Gateway to support cross-cluster traffic.
+# This feature is supported only with encap mode.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "Multicluster" "default" false) }}
+
+# Enable support for provisioning secondary network interfaces for Pods (using
+# Pod annotations). At the moment, Antrea can only create secondary network
+# interfaces using SR-IOV VFs on baremetal Nodes.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "SecondaryNetwork" "default" false) }}
+
+# Enable managing external IPs of Services of LoadBalancer type.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "ServiceExternalIP" "default" false) }}
+
+# Enable mirroring or redirecting the traffic Pods send or receive.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "TrafficControl" "default" false) }}
+
+# Enable certificated-based authentication for IPsec.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "IPsecCertAuth" "default" false) }}
+
+# Name of the primary OpenVSwitch bridge antrea-agent will create for Secondary interface needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-int1.
+Secondary_OVSBridge1: {{ (.Values.secondaryNetworkOVS).bridgeName1 | quote }}
+
+# Name of the OpenVSwitch bridge antrea-agent will create for Secondary network specific tunnel needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-inti2.
+Secondary_OVSBridge2: {{ (.Values.secondaryNetworkOVS).bridgeName2 | quote }}
+
+# Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
+# - system
+# - netdev
+# 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
+# OVS in userspace mode. Userspace mode requires the tun device driver to be available.
+Secondary_OVSDatapathType: {{ (.Values.secondaryNetworkOVS).ovsDatapathType | quote }}
+
+# Name of the OVS patch port which would connected two OVS bridges per secondary inteface config.
+Secondary_OVSPatchPort: {{ (.Values.secondaryNetworkOVS).ovsPatchPort | quote }}
+
+# Name of the OVS patch port peer which would connected two OVS bridges per secondary inteface config.
+Secondary_OVSPatchPortPeer: {{ (.Values.secondaryNetworkOVS).ovsPatchPortPeer | quote }}
```

Comment:
At least let us generate the peer names from a base name. We may not want to expose so many detailed parameters.

Reply:
Aligned and made changes accordingly.